docs/jobs-api.md

```diff
@@ -285,3 +285,20 @@ GET /api/v0/paths/forever.juttle HTTP/1.1
     }
 }
 ```
+
+###GET /version
```

Comment:
I somehow thought that would make it look like version info of the `/api` itself as opposed to components the server is running on. Don't have a strong preference though. Will change.